src/test/java/uk/nickbdyer/chatserver/ChatServerTest.java

```diff
@@ -0,0 +1,21 @@
+package uk.nickbdyer.chatserver;
+
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertTrue;
+
+public class ChatServerTest {
+
+    @Test
```

Comment:
London style mock test. This behaviour is now captured via Chicago style integration style tests. Should I delete this, or is there value in keeping it.